### PR TITLE
Fixing things

### PR DESCRIPTION
**Pull request by timothycobrien**
Maybe squash some of these commits lol

### CODE REVIEW

**Comment by timothycobrien** [comment]
> You should rebase on my master.
> 
> ```
> git remote add upstream https://github.com/ayazhafiz/gradec
> git fetch upstream
> git rebase upstream/master
> ```

**Comment by timothycobrien** [comment]
I really thought I did before, but I guess not? Should be good now

File: src/grader.ts [hunk 1]
```diff
@@ -245,11 +245,11 @@ export class Grader implements GradeHandleIterator {
...
       async function calculateAndPostGrade(): Promise<api.CommentScoreResult> {
         const score = await scoreComments(requestMeta);
-        const scoreStr = score === MAXSCORE ? '💯' : `${score}/${MAXSCORE}`;
-        const finalScoreComment = `${SCORE_PREFIX} ${scoreStr}`;
+        const finalScoreComment =
+        `${SCORE_PREFIX} ${(score === 100) ? ('💯') : (score + '/' + MAXSCORE)}`;
         const postingResult = await postComment(requestMeta, finalScoreComment);
 
-        return {comment: finalScoreComment, score, url: postingResult.html_url};
+        return { comment: finalScoreComment, score, url: postingResult.html_url };
```
Comment (ayazhafiz) [inline src/grader.ts:252]: These changes shouldn't be made. Are you sure you fetched my master branch and rebased on it?

Comment (timothycobrien) [reply src/grader.ts:252]: Yea but I might have merged the wrong branch by accident, though i'm pretty sure I just took remote?

Comment (ayazhafiz) [reply src/grader.ts:252]: try

```
git fetch upstream
git rebase upstream/master

**Comment by ayazhafiz** [comment]
This is stale; closing. @timothycobrien feel free to open a new PR if needed.

**Pull request closed without merging**